addOns/network/network.gradle.kts

```diff
@@ -0,0 +1,32 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+description = "Provides networking capabilities."
+
+zapAddOn {
+    addOnName.set("Network")
+    addOnStatus.set(AddOnStatus.ALPHA)
+    zapVersion.set("2.12.0")
+
+    manifest {
+        author.set("ZAP Dev Team")
+        url.set("https://www.zaproxy.org/docs/desktop/addons/network/")
+
+        helpSet {
+            baseName.set("help%LC%.helpset")
+            localeToken.set("%LC%")
+        }
+    }
+}
+
+crowdin {
+    configuration {
+        tokens.put("%messagesPath%", "org/zaproxy/addon/${zapAddOn.addOnId.get()}/resources/")
+        tokens.put("%helpPath%", "")
+    }
+}
+
+dependencies {
+    zap("org.zaproxy:zap:2.11.0")
```

Comment:
Does it matter that this mismatches with Line 8?

Reply:
It's on purpose, we need to compile with 2.11 for the time being (the add-on is meant for 2.12).

Reply:
Okay, perfect, just wanted to be sure.